### DISCUSSION TITLE
Link to on same page to #id in app directory hit or miss

### DISCUSSION THREAD

**Discussion by ctrenks** [vercel/community#3940] [Help]
### Page to Investigate

https://afc-three.vercel.app/review/jackpot-capital

### Steps to Reproduce

Click on a "Jump To" link and the page refreshes like clicking an <a> link to the page.  I tried both <a> and <Link> with same results.  This feature does work on some pages but not others.  If you copy the href link from elements it work in browser as expected.  I did see this was an issue in 2021 but appears to be causing me an issue now,

**Comment by ctrenks** [comment]
This is not an issue while running locally btw.

**Comment by amyegan** [comment]
It looks like you might have a typo preventing the links from jumping to the right location on the page. For example, 
`<a href="#ProsCons">Jackpot Capital Pros and Cons</a>` is looking for an element with the id value `ProsCons`. But the element that seems like your target, `<div id="proscons"...`, doesn't have matching capitalization. If you fix that, I think it will solve your problem.

**Comment by ctrenks** [comment]
No that was me playing after the fact changing all to lowercase but that is not the issue.  Let me finish changing them all to lc

**Comment by ctrenks** [comment]
So for some reason they are working now, when i posted this not one workied on this page

**Reply by amyegan** [reply]
Glad it's working now! Please let me know if it breaks again. I'm interested to know what actually caused the problem.
